src/keystore.cpp

```diff
@@ -149,20 +147,16 @@ bool CBasicKeyStore::AddSproutViewingKey(const libzcash::SproutViewingKey &vk)
     return true;
 }
 
-bool CBasicKeyStore::AddSaplingFullViewingKey(
+void CBasicKeyStore::AddSaplingFullViewingKey(
```

Comment:
Leave this as returning `bool`, since `AddSaplingIncomingViewingKey` can fail (once it also returns `bool`).